pro/src/context/IndividualOfferContext/IndividualOfferContext.tsx

```diff
@@ -117,12 +120,15 @@ export function IndividualOfferContextProvider({
         setSubCategories(response.payload.categoriesData.subCategories)
         setOffererNames(response.payload.offererNames)
         setVenueList(response.payload.venueList)
-        const venuesPopinDisplaying: Record<string, boolean> = {}
-        response.payload.venueList.forEach((v) => {
-          venuesPopinDisplaying[v.id] =
-            !v.hasCreatedOffer && v.hasMissingReimbursementPoint
-        })
-        setShowVenuePopin(venuesPopinDisplaying)
+
+        if (!isNewBankDetailsJourneyEnabled) {
+          const venuesPopinDisplaying: Record<string, boolean> = {}
+          response.payload.venueList.forEach((v) => {
+            venuesPopinDisplaying[v.id] =
+              !v.hasCreatedOffer && v.hasMissingReimbursementPoint
+          })
+          setShowVenuePopin(venuesPopinDisplaying)
```

Comment:
suggestion: nommer la condition
+ utiliser un for of

tu trouves ça mieux ?

```suggestion
const venuesPopinDisplaying: Record<string, boolean> = {}

for (const venue of response.payload.venueList) {
  const shouldDisplayPopin = !venue.hasCreatedOffer && venue.hasMissingReimbursementPoint
  venuesPopinDisplaying[venue.id] = shouldDisplayPopin
}

setShowVenuePopin(venuesPopinDisplaying)
```

Reply:
On peut, c'est du code qui va disparaitre mais c'est mieux oui